### ISSUE TITLE
How to use model1.py?

### ISSUE THREAD

**Issue by wenku1992abc** [xuanjihe/speech-emotion-recognition#44]
when i use: python model1.py
there is such errors:
![图片](https://user-images.githubusercontent.com/33471245/103184573-bbcd9f80-48f3-11eb-91d1-04310610d5a5.png)

**Comment by xuanjihe** [comment]
your logits outputs 6 classes, but your input labels only 4 classes
